conformance/resources/manifests/manifests.yaml

```diff
@@ -42,7 +50,7 @@ spec:
     protocol: HTTP
     allowedRoutes:
       namespaces:
-        from: Same # Restrict to same namespace initially for simplicity
+        from: All 
```

Comment:
why changing to all? are we testing cross-ns routes?

Reply:
I changed it since the shared Gateway in `gateway-conformance-infra` must accept routes from test namespaces e.g., `gateway-conformance-app-backend`.